### PR TITLE
update : sign up 

### PR DESCRIPTION
**Pull request by Bibin-Antony**

### CODE REVIEW

**Comment by hrishikeshmane** [comment]
Looks good. You can merge.

File: html/signUp.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                    </svg>
+                </div>
+                <p class="left--content">
+                    Our customers are our gretest priority which hence our unique bussiness model.
+                </p>
+                <div class="footer-left--payment">
+                    <img src="/assests/image/rupay.png" alt="">
+                    <img src="/assests/image/visa.png" alt="">
+                    <img src="/assests/image/Paypal-logo.png" alt="">
+                    <img src="/assests/image/mastercard.png" alt="">
```
Comment (Aditi0505) [inline html/signUp.html:137]: Suggestion: Please provide alt description for accessibility checkpoint.

File: styles/index.css [hunk 2]
```diff
@@ -31,18 +32,15 @@ body {
...
     --white-color: white;
     --nav-floating: #ff4141;
-
     --footer-color: #111;
     --light-text: #9e9e9e;
     --transparent-layout: #c8c8c866;
-
     --footer-color: #111;
     --light-text: #9e9e9e;
     --transparent-layout: rgba(200, 200, 200, 0.4);
```
Comment (Aditi0505) [inline styles/index.css:40]: Suggestion: You can convert this rgba value to hex (for consistency) using online tools.

**Review by Aditi0505 — APPROVED** [review]
Left some suggestions. Rest looks good to me. 
Also, it would be good to provide description and screenshots as well for better visualization.

File: html/signUp.html (hunk 1 above)
Comment (Bibin-Antony) [reply html/signUp.html:137]: ok will do

File: styles/index.css (hunk 2 above)
Comment (Bibin-Antony) [reply styles/index.css:40]: will do

**Pull request merged**